.travis.yml

```diff
@@ -60,13 +60,12 @@ jobs:
         - set -o errexit
         - flake8
         - ./clang-format-diff.sh
+        - ./clang-tidy-diff.sh
```

Comment:
Can clang-tidy run before cmake? It requires a compilation database file (`compile_commands.json`) generated by cmake.